legacy/debug_signing/README.md

```diff
@@ -0,0 +1,77 @@
+# Debugging T1 signatures
```

Comment:
This is supposed to be a "separate chapter" about signatures debugging with documentation (`scripts` is more regarding build). Originally this contained also examples of unsigned and signed trezor.bin (both styles) and other outputs, which were made by hand in hex editor.
But in the end I decided not to add the trezor.bin files to not inflate the size of the repo, since now they can be created rather effortlessly as compared to before.